src/javaRosa.js

```diff
@@ -1452,9 +1331,10 @@ define([
         loadXML: function (xmlString) {
             var _this = this,
                 langs = this.opts().javaRosa.langs,
-                Itext;
+                Itext, itextMap;
 
             this.data.javaRosa.Itext = Itext = new ItextModel();
+            this.data.javaRosa.itextMap = itextMap = {};
```

Comment:
Thinking about this more, I think my "one possible issue" above is not actually an issue because I think there was/is no way to associate a mug itext property with an existing (possibly unreferenced) itext item aside from manually editing the XML, particularly because of the way [deduplication of itext IDs](https://github.com/dimagi/Vellum/pull/360/files#r26888162) is done.